lib/Settings/SettingsLabel/SettingsLabel.js

```diff
@@ -16,15 +17,16 @@ const SettingsLabel = ({
         app={app}
         iconKey={iconKey}
       >
-        <FormattedMessage id={messageId} />
+        {messageId ? <FormattedMessage id={messageId} /> : message}
```

Comment:
Please, could you clarify what case will be covered by this change?

Reply:
On the details page, there is a need to show the profile name with the icon.

Reply:
```suggestion
        {messageId && <FormattedMessage id={messageId} /> }
        {children}
```

Reply:
Based on the discussion with decided to proceed with `children` approach.